index.html

```diff
@@ -4,49 +4,371 @@
+            </div>
+            <div class="image_geometry_2">
+                <img src="images/image_geometry_2.png" alt="">
+            </div>
+        </div>
+        <!-- end mobile version -->
+
+        <div class="intro">
+            <h1>Leo de Andrade</h1>
+            <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
+            <button title="Start Collaboration" class="btnstart" onclick=" window.open('#get-started', '_self'); return false;">Start Collaboration</button>
         </div>
         <div class="dots">
-            <img src="images/image_dots.png" alt="image_dots.png">
+            <img src="images/image_dots.svg" alt="">
+        </div>
+        <div class="image-geometri1">
+            <img src="/images/image-geometry_1.svg" alt="">
+        </div>
+    </section>
+
+    <!-- Projects Section -->
+
+    <!-- mobile version start -->
+    <div class="mobile-project">
+
+        <div class="mobile-project-tit">
+            <h1 class="mobile-projecttit">Projects</h1>
+        </div>
+        <div class="mobile-project1">
+            <img src="images/mobile-Rectangle21.png" alt="">
+                <div class="mobile-ret21">
+                    <h3 class="ret21">Project name goes here</h3>
+                    <ul class="overdescription">
+                        <li><a href="#">HTML/CSS</a></li>
+                        <li><a href="#">Ruby on Rails</a></li>
+                        <li><a href="#">JavaScript</a></li>
+                    </ul>
+                <button title="See this project" class="btnseethis" onclick=" window.open('#get-started', '_self'); return false;">See this project</button>
+            </div>
+        </div>
+        <div class="mobile-retangule22">
+            <img src="images/mobile-Rectangle22.png" alt="">
+            <div class="mobile-ret21">
+                <h3 class="ret21">Project name goes here</h3>
+                <ul class="overdescription">
+                    <li><a href="#">HTML/CSS</a></li>
+                    <li><a href="#">Ruby on Rails</a></li>
+                    <li><a href="#">JavaScript</a></li>
+                </ul>
+            <button title="See this project" class="btnseethis" onclick=" window.open('#get-started', '_self'); return false;">See this project</button>
+            </div>    
+        </div>
+
+        <div class="mobile-retangule25">
+            <img src="images/mobile-Rectangle25.png" alt="">
+            <div class="mobile-ret21">
+                <h3 class="ret21">Project name goes here</h3>
+                <ul class="overdescription">
+                    <li><a href="#">HTML/CSS</a></li>
+                    <li><a href="#">Ruby on Rails</a></li>
+                    <li><a href="#">JavaScript</a></li>
+                </ul>
```

Comment:
Optional suggestions but I highly recommend you to work on it before you merge, if you need any help ping on slack.

- [ ] Kindly, add the `text-decoration: none` in your class that holds `HTML/CSS RoR and JavaScript section` to look the same as the design on Figma.
- [ ] Kindly, add the hover color to your `see project` button